packages/suite/src/views/wallet/tokens/components/TokenList.tsx

```diff
@@ -68,92 +76,134 @@ const StyledNoRatesTooltip = styled(NoRatesTooltip)`
     justify-content: flex-end;
 `;
 
+const StyledQuestionTooltip = styled(QuestionTooltip)`
+    margin-top: ${spacingsPx.xxl};
+    margin-bottom: ${spacingsPx.sm};
+`;
+
 interface TokenListProps {
     tokens: Account['tokens'];
     networkType: Account['networkType'];
     explorerUrl: string;
     explorerUrlQueryString: string;
     isTestnet?: boolean;
+    symbol: Account['symbol'];
```

Comment:
nit: for better readibility

```suggestion
    networkSymbol: NetworkSymbol;
```

Reply:
23fb1f90fcfd1b7599914eaa492bf5a4b0d2058b